services/api/src/resources/insight/resolvers.ts

```diff
@@ -0,0 +1,162 @@
+import * as R from 'ramda';
+import { sendToLagoonLogs } from '@lagoon/commons/dist/logs';
+import { ResolverFn } from '..';
+import {
+  pubSub,
+  createEnvironmentFilteredSubscriber
+} from '../../clients/pubSub';
+import { getConfigFromEnv, getLagoonRouteFromEnv } from '../../util/config';
+
+import { getEnvironmentName, convertBytesToHumanFileSize } from './helpers';
+import { Helpers as environmentHelpers } from '../environment/helpers';
+import { Helpers as projectHelpers } from '../project/helpers';
+
+import S3 from 'aws-sdk/clients/s3';
+
+// s3 config
+const accessKeyId =  process.env.S3_FILES_ACCESS_KEY_ID || 'minio'
+const secretAccessKey =  process.env.S3_FILES_SECRET_ACCESS_KEY || 'minio123'
+const bucket = process.env.S3_INSIGHTS_BUCKET || 'lagoon-insights'
```

Comment:
The insights handler just assumes that this is the S3_FILES_BUCKET - maybe we should here too?